app/core/Router.coffee

```diff
@@ -176,10 +176,10 @@ module.exports = class CocoRouter extends Backbone.Router
     'play/level/:levelID': go('play/level/PlayLevelView')
     'play/game-dev-level/:sessionID': go('play/level/PlayGameDevLevelView')
     'play/web-dev-level/:sessionID': go('play/level/PlayWebDevLevelView')
-    'play/game-dev-level/:levelID/:sessionID': (levelID, sessionID) ->
-      @navigate("play/game-dev-level/#{sessionID}", { trigger: true, replace: true })
-    'play/web-dev-level/:levelID/:sessionID': (levelID, sessionID) ->
-      @navigate("play/web-dev-level/#{sessionID}", { trigger: true, replace: true })
+    'play/game-dev-level/:levelID/:sessionID': (levelID, sessionID, queryString) ->
+      @navigate("play/game-dev-level/#{sessionID}?#{queryString}", { trigger: true, replace: true })
```

Comment:
What is queryString is undefined or null?

Reply:
In that case, it will send `?null` as the query in the URL. It doesnt cause any problems as of now, but do you suggest we should handle it separately anyways?